starters/vue3-apollo-quasar/src/layouts/MainLayout.vue

```diff
@@ -36,8 +33,15 @@ useMeta({
 </script>
 
 <style lang="scss">
+.page-header {
+  width: 100%;
+  margin: 20px auto;
+  @media (min-width: 1024px) {
```

Comment:
try to avoid media. in custom styles

Reply:
What do you propose here?

Reply:
The workaround is to set max-width in **em**. I think it can be more responsive and pretty for any screen width